src/ImageSharp/Formats/Webp/Lossy/Vp8Encoder.cs

```diff
@@ -405,16 +406,23 @@ public void Encode<TPixel>(Image<TPixel> image, Stream stream)
         ExifProfile exifProfile = this.skipMetadata ? null : metadata.ExifProfile;
         XmpProfile xmpProfile = this.skipMetadata ? null : metadata.XmpProfile;
 
-        this.bitWriter.WriteEncodedImageToStream(
-            stream,
-            exifProfile,
-            xmpProfile,
-            metadata.IccProfile,
-            (uint)width,
-            (uint)height,
-            hasAlpha,
-            alphaData[..alphaDataSize],
-            this.alphaCompression && alphaCompressionSucceeded);
+        try
+        {
```

Comment:
your going to want to start this `try` before the `if (hasAlpha)` line otherwise anything that fails before the `WriteEncodedImageToStream` line will cause the memory to leak